app/src/main/res/values/colors.xml

```diff
@@ -152,4 +152,7 @@
   <color name="onboarding3StatusBar">#AD8500</color>
   <color name="onboarding4StatusBar">#3B5893</color>
   <color name="registrationStatusBar">#9CB1BE</color>
+  <!-- ICON COLORS -->
+  <color name="iconDefault">#FF000000</color>
```

Comment:
These colors should be specific to where they're being used, e.g. ``mergeIconEnabled`` and ``mergeIconDisabled``

Reply:
done